example/scalalib/linting/2-contrib-scoverage/build.mill

```diff
@@ -14,6 +14,9 @@ object `package` extends ScoverageModule {
     mvn"com.lihaoyi::mainargs:0.7.7"
   )
 
+  def statementCoverageMin = Some(10.00)
```

Comment:
Let's set this to 15, so we can show how a failure looks like.

Reply:
You can use `sed` to modify the files during the test. Grep for other examples to see how it's done